### PR TITLE
chore(issue templates) update wording in templates

### PR DESCRIPTION
**Pull request by ArjunSharda**
## Changes
This will update the wording in the templates as per what I think should be okay.

## How Has This Been Tested?
Not tested; wording changes

## Documentation
N/A - Grammatical changes

### CODE REVIEW

**Comment by ArjunSharda** [comment]
Please review when possible @adrians5j

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: "\U0001F41B Bug report"
-description: Create a report to help us improve
+description: Report a issue that you have spotted
 body:
```
Comment (swapnilmmane) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:3]: Hi @ArjunSharda, thank you for the PR. 
Could you please change the  `Report a issue` to `Report an issue`

File: .github/ISSUE_TEMPLATE/2-feature-request.yml [hunk 2]
```diff
@@ -8,12 +8,11 @@ body:
         Please fill in as much of the template below as you're able.
   - type: textarea
     attributes:
-      label: Is your feature request related to a problem? Please describe.
-      description: Please describe the problem you are trying to solve.
+      label: Is your feature request related to a problem? Please describe if so.
```
Comment (swapnilmmane) [inline .github/ISSUE_TEMPLATE/2-feature-request.yml]: @ArjunSharda, it seems the previous label and description are good. Let's keep it as it is.

Comment (ArjunSharda) [reply .github/ISSUE_TEMPLATE/2-feature-request.yml]: ok

File: .github/ISSUE_TEMPLATE/2-feature-request.yml [hunk 3]
```diff
@@ -5,15 +5,14 @@ body:
     attributes:
       value: |
         Thank you for suggesting an idea to make Webiny better.
-        Please fill in as much of the template below as you're able.
+        Please fill in as much of the template below as you're able to.
   - type: textarea
     attributes:
       label: Is your feature request related to a problem? Please describe.
-      description: Please describe the problem you are trying to solve.
```
Comment (swapnilmmane) [inline .github/ISSUE_TEMPLATE/2-feature-request.yml:12]: Sorry @ArjunSharda, if it was not clear in my previous comment. I would like to keep `description` as well. Let's not remove it.

Comment (swapnilmmane) [reply .github/ISSUE_TEMPLATE/2-feature-request.yml:12]: Hi @ArjunSharda, it seems you mistakenly marked this comment as resolved. 
I am still able to see this line is deleted
> description: Please describe the problem you are trying to solve.

Could you please re-add it your repository, thank you!

**Pull request merged**